de-lib/src/main/java/org/iplantc/de/theme/base/client/analyses/AnalysesMessages.properties

```diff
@@ -52,3 +52,5 @@ supportRequestSuccess = Your support request is submitted. A support staff will
 userRequestingHelpSubject = requesting help with Analysis
 requestProcessing = Processing your request...
 stepInfoDialogHeader = Analysis Info
+interactiveAnalysisTooltip= Go to VICE analysis url
+
```

Comment:
May be `Access your interactive analyses` ? Might want to check with Science team or even Nirav. We haven't used `VICE` in app type filter.

Reply:
Oh good point, I didn't think about the filter names.  I'll update it to match 👍